spec/src/consensus.rs

```diff
@@ -191,13 +191,16 @@ pub fn build_genesis_epoch_ext(
     compact_target: u32,
     genesis_epoch_length: BlockNumber,
     epoch_duration_target: u64,
+    genesis_orphan_rate: (u32, u32),
 ) -> EpochExt {
     let block_reward = Capacity::shannons(epoch_reward.as_u64() / genesis_epoch_length);
     let remainder_reward = Capacity::shannons(epoch_reward.as_u64() % genesis_epoch_length);
 
-    let genesis_orphan_count = genesis_epoch_length / 40;
+    // let genesis_orphan_count = genesis_epoch_length * genesis_orphan_rate.0 / genesis_orphan_rate.1;
```

Comment:
code cleanup: please remove comment-out code

Reply:
I keep this line as a comment, since the followed formula has been simplified.
And there really are someone who has asked me for that.